libs/design/paginator/src/paginator/paginator.component.html

```diff
@@ -2,15 +2,15 @@
   <button type="button" class="daff-paginator__previous"
     [disabled]="_disablePrev"
     tabindex="0"
-    attr.aria-label="Go to Previous Page of {{_paginatorId}} Paginator"
+    attr.aria-label="Go to Previous Page of {{ariaLabel}} Paginator"
```

Comment:
I was looking at various paginator implementations and I believe that adding the label in here is non-normative.

https://a11ymatters.com/pattern/pagination/
https://design-system.w3.org/components/pagination.html

I believe this should solely read: "Go to Previous Page".

@xelaint